File: h2o-docs/src/booklets/v2_2015/source/GLM_Vignette_code_examples/glm_ordinal_example.py
import h2o
from h2o.estimators.glm import H2OGeneralizedLinearEstimator
h2o.init()
h2o_df = h2o.import_file("http://h2o-public-test-data.s3.amazonaws.com/bigdata/laptop/glm_ordinal_logit/ordinal_multinomial_training_set.csv")
h2o_df['C11'] = h2o_df['C11'].asfactor()

ordinal_fit = H2OGeneralizedLinearEstimator(family = "ordinal", alpha = 1.0, lambda_=0.000000001, obj_reg = 0.00001, max_iterations=1000, beta_epsilon=1e-8, objective_epsilon=1e-10)
ordinal_fit.train(x=list(range(0,10)), y="C11", training_frame=h2o_df)

File: h2o-docs/src/booklets/v2_2015/source/GLM_Vignette_code_examples/glm_ordinal_example.R
library(h2o)
h2o.init()
h2o_df = h2o.import_file("http://h2o-public-test-data.s3.amazonaws.com/bigdata/laptop/glm_ordinal_logit/ordinal_multinomial_training_set.csv")
Dtrain$C11 <- h2o.asfactor(Dtrain$C11)
X <- c(1:10)  
Y <-"C11"
ordinal.fit <- h2o.glm(y = Y, x = X, training_frame = Dtrain, lambda=c(0.000000001), alpha=c(0.7), family = "ordinal", beta_epsilon=1e-8, objective_epsilon=1e-10, obj_reg=0.00001,max_iterations=1000 )
